examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -102,19 +106,9 @@ angular.module('cloudberry.common', [])
           values: keywords
         }
       ];
-      if (parameters.geoIds.length <= 2000){
-        filter.push(
-          {
-            field: "geo_tag." + spatialField,
-            relation: "in",
-            values: parameters.geoIds
-          }
-        );
-      }
-      return filter;
```

Comment:
These parts should not be removed, if it's currently in the master. These parts were introduced by another request.